use-api.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: docs
+title:  "How to Use the API"
+---
+
+## Access methods 
+The API is public and available for GET requests at [https://api.rockarch.org](https://api.rockarch.org). 
+
+Get started with your favorite API tool or script. We do not require an API key.
+
+- If you are new to scripting or working with APIs, consider using a tool like [Hoppscotch](https://hoppscotch.io/) or [Postman](https://www.postman.com/).
+- Browse the API online at [https://api.rockarch.org](https://api.rockarch.org).
+- We provide an [API client](https://pypi.org/project/rac-api-client/) to simplify requests if you are writing Python scripts.
+
+
+## Quick start 
```

Comment:
some brief contextual information describing the purpose of this section and how readers can use it may be helpful - especially for distinguishing it from the example queries section